source/extensions/filters/network/rbac/rbac_filter.cc

```diff
@@ -10,6 +10,10 @@ namespace Extensions {
 namespace NetworkFilters {
 namespace RBACFilter {
 
+static const std::string engine_result_allowed = "allowed";
```

Comment:
same for these strings as well (and can you also do singleton for http filter ones?)

Reply:
Addressed in fcd1eab.